receiver/lokireceiver/go.mod

```diff
@@ -32,7 +32,6 @@ require (
 )
 
 require (
-	cloud.google.com/go/compute/metadata v0.2.4-0.20230617002413-005d2dfb6b68 // indirect
```

Comment:
this change can be reverted.

Reply:
this has been removed by `make gotidy`
I have reverted this change 
I have also reverted all other changes done by `make gotidy` in lokireceiver and opencensusreceiver